bin/run-tests.sh

```diff
@@ -13,8 +13,6 @@
 
 exit_code=0
 
-sbt test
-
```

Comment:
`sbt assembly` in the Dockerfile already run the tests, so this line causes the tests to be run twice when runnning `run-tests-in-docker.sh`.

I have a more generic question: are `run.sh` and `run-tests.sh` supposed to work outside a docker image? If so, both are broken in this PR and I'll investigate. That being said, those files don't seem to work for me outside docker on the `main` branch as well.

Reply:
> sbt assembly in the Dockerfile already run the tests, so this line causes the tests to be run twice when runnning run-tests-in-docker.sh.

Ah, I would not have guessed that `sbt assembly` runs the tests too. Then it is fine to omit it here.

> I have a more generic question: are run.sh and run-tests.sh supposed to work outside a docker image? If so, both are broken in this PR and I'll investigate. That being said, those files don't seem to work for me outside docker on the main branch as well.

Ehm, not necessarily. For some tracks, they can. I wouldn't be opposed to a PR that makes them work outside the docker image, but only if it can be done in a somewhat sane way :)

Reply:
I would say let's leave it as is for now, then :zipper_mouth_face: